zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -521,33 +559,31 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
             tx.value,
             gas_fee,
         )?;
-        self.phase2_code_hash
-            .assign(region, offset, region.word_rlc(callee_code_hash))?;
-        self.is_empty_code_hash.assign_value(
+        // TODO handle endian issue
+        self.code_hash
+            .assign_u256(region, offset, callee_code_hash)?; // big endian assignment
```

Comment:
Is this TODO resolved?

Reply:
Here the type is U256, but it's big endian. The reason is because we implement `ToWord` for `Hash` without align to little endian so cause confusing. I try to fixed in  https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1411/commits/5f0bdfc85f79ef835cafded98df8d76005251822 and unittest still pass (just verify test under evm circuit) Do you think above fixed make sense~?

Reply:
Fixing in another PR makes sense. Can we create an issue to track this and add the issue number in the TODO note?

Reply:
issue raised, https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/1467 and rollback fixed in this pr to limit the scope